frontend/src/components/yugioh/table/market/AddToCardButton.tsx

```diff
@@ -0,0 +1,25 @@
+import { Button } from '@mui/material';
+import ShoppingCartIcon from '@mui/icons-material/ShoppingCart';
+
+function AddToCardButton(): JSX.Element {
+  return (
+    <Button
+      color="success"
+      sx={{
+        backgroundColor: '#15B58D',
+        borderColor: '#3A3A3A',
+        color: 'white',
+        ':hover': {
+          color: 'black',
+        },
+      }}
+      aria-label="Add to cart"
+      size="small"
+      className="border h-12 w-14"
```

Comment:
- [ ] `AddToCard` -> `AddToCart`
- [ ] Make the height of the button `h-10`

Reply:
This is done